hal/armv7a/arch/cpu.h

```diff
@@ -207,7 +207,7 @@ static inline unsigned int hal_cpuGetCount(void)
 }
 
 
-static inline void cpu_sendIPI(unsigned int cpu, unsigned int intr)
+static inline void cpu_broadcastIPI(unsigned int intr)
```

Comment:
This empty ARM prototypes are redundant - it's already defined in hal/cpu.h as non-static and non-inline. Empty implementations (in C) should be provided instead